src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  text-transform: uppercase;
+}
+
+.logo_link {
+  height: 40px;
+  width: 40px;
+}
+
+.navlink {
```

Comment:
use the same code styling, you've had `logo_link`, so rename `navlink` -> `nav_link`